kotlin-analysis/src/main/kotlin/org/jetbrains/dokka/analysis/AnalysisEnvironment.kt

```diff
@@ -218,6 +215,15 @@ class AnalysisEnvironment(val messageCollector: MessageCollector, val analysisPl
             override val platform: TargetPlatform = targetPlatform
             override fun dependencies(): List<ModuleInfo> =
                 listOf(this, library) + extraModuleDependencies
+
+            /**
+             * Only for common platform ignore BuiltIns for StdLib since it can cause a conflict
+             * between BuiltIns from a compiler and ones from source code.
+             */
```

Comment:
nitpick: even though it's unlikely that this will be used for anything other than stdlib, mentioning `StdLib` in KDoc can be confusing, since the boolean `ignoreCommonBuiltIns` can be set for any project

and I think the comment can be moved to `DokkaAnalysisConfiguration` - I think people will mostly look for the explanation there